src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
I'm kind of concerned with what would happen with unknown at-rules. There many at-rules that will still break things: vendor prefixed rules (`@-moz-keyframes`), obscure rules (`@document`), garbage/misspellings (`@improt`), or anything that the W3C cooks up next.

In these situations it would probably be best if the at-rule was ignored and a warning was issued. Crashing with a TypeError error is probably not the most user friendly way of letting people know we missed something. :smile: